### PR TITLE
Update to fedora 25 plus misc improvements

### PR DESCRIPTION
**Pull request by Raffarti**
Hi there,
I happened to use this for a koha installation (quite convenient indeed).
Here some changes I made so it can run smoothly on Fedora 25, see if you can save something.
Tested on 32 bit only.

### CODE REVIEW

**Comment by nemobis** [comment]
Thanks, I was surprised and very glad that this was of some use for you. :) Are you still working on it?

I might fire up a virtual machine tomorrow and test it quickly.

**Comment by Raffarti** [comment]
I'm concerned by 2efb7e1, SELinux was blocking catalog search with zebra. Is zebra_custom.pp there for the same reason, but it's not working?

**Comment by nemobis** [comment]
Raffarti, 20/03/2017 23:59:
> I'm concerned by 2efb7e1
> <https://github.com/nemobis/beic-koha/commit/2efb7e1f40d7b34343e53841b662e5a63e095833>,
> SELinux was blocking catalog search with zebra. Is zebra_custom.pp there
> for the same reason, but it's not working?

I don't understand the question. SELinux was working correctly for me. 
Did you check that /tmp/zebra_custom.pp existed? If it existed, was the 
step "SELinux, install module to allow zebradb socket access" executed? 
If it was executed, do output or logs say anything about the outcome?

**Comment by Raffarti** [comment]
zebra_custom shows up in `semodule -l|grep zebra`, but SELinux is still blocking search.pl.

[selinux_exception.txt](https://github.com/nemobis/beic-koha/files/857616/selinux_exception.txt)
[my-searchpl.te](https://github.com/nemobis/beic-koha/files/857617/my-searchpl.txt)

**Comment by Raffarti** [comment]
Probably the old policy just don't fit fedora 25.

**Comment by Raffarti** [comment]
I've finally managed to fix zebra search! Cheers.

**Comment by nemobis** [comment]
I'm finally testing this, on fedora 25 with Koha 17.05, and there seem to be some (probably) minor issues such as a couple dependencies. I hope to have follow-up commits in short order to confirm this works.

**Comment by nemobis** [comment]
One dependency error vanished out of the blue and the other could only be worked around, more details in https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=19717

File: ansible-koha/files/zebraidx.service [hunk 1]
```diff
@@ -1,12 +1,14 @@
 [Unit]
 Description=Zebra daemon for Koha
+Requires=mariadb.service
+After=mariadb.service
 
 [Service]
 Type=simple
 User=koha
 PermissionsStartOnly=true
-ExecStartPre=/usr/bin/chmod -R 777 /var/lock/koha/zebradb/ /var/run/koha/zebradb/ /var/lib/koha/zebradb/
```
Comment (nemobis) [inline ansible-koha/files/zebraidx.service:8]: This was needed! I'm moving it to its own task. I was wondering where all those zebra failures were coming from. :)

**Comment by nemobis** [comment]
Thanks! Now I verified it works. Sorry it took so long... I was not running a Koha machine these past months.

**Comment by Raffarti** [comment]
A pleasant surprise to see it finally in shape! Installing Koha can really be frustrating w/o this.

**Pull request merged**